pkg/controller/core.oam.dev/v1alpha2/applicationconfiguration/suite_test.go

```diff
@@ -150,7 +151,9 @@ var _ = BeforeSuite(func(done Done) {
 	}, time.Second*30, time.Millisecond*500).Should(BeNil())
 	Expect(mapping.Resource.Resource).Should(Equal("foo"))
 
-	reconciler = NewReconciler(mgr, dm, WithLogger(logging.NewLogrLogger(ctrl.Log.WithName("suit-test-appconfig"))))
+	reconciler = NewReconciler(mgr, dm,
+		WithLogger(logging.NewLogrLogger(ctrl.Log.WithName("suit-test-appconfig"))),
+		WithApplyOnceOnlyMode(core_oam_dev.ApplyOnceOnlyOff))
```

Comment:
make it as default in NewReconciler, so you don't need to add this line

Reply:
got it